src/install/install.zig

```diff
@@ -7639,7 +7657,15 @@ pub const PackageManager = struct {
             // add
             // remove
             outer: for (positionals) |positional| {
-                var input = std.mem.trim(u8, positional, " \n\r\t");
+                var input: []u8 = @constCast(std.mem.trim(u8, positional, " \n\r\t"));
+                {
+                    var temp: [2048]u8 = undefined;
+                    const len = std.mem.replace(u8, input, "\\\\", "/", &temp);
```

Comment:
this should be `input[0..temp.len]` to be safe

Reply:
what would be the best way to surface an error? ideally if `input.len` > `temp.len` it should fail